helpdesk/management/commands/change_emails.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+
+from django.contrib.auth.models import User
+
+from helpdesk.models import Ticket
+
+class Command(BaseCommand):
+    help = 'Switches all tickets with an old email address to a new one.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('old_email', type=str)
+        parser.add_argument('new_email', type=str)
+
+    def handle(self, *args, **options):
+        """
+        Get all tickets with old email and update them with new email.
+        """
+
+        old_email = options['old_email']
+        new_email = options['new_email']
+
+        try:
+            User.objects.get(email=new_email)
+        except ObjectDoesNotExist:
+            raise CommandError('%s is not valid for any user' % new_email)
+
+        old_tickets = Ticket.objects.filter(submitter_email=old_email)
+
+        self.stdout.write('%s has %i tickets' %
+            (old_email, len(old_tickets)))
```

Comment:
This line (and others like it) should be indented again. See following flake8 errors:

```
helpdesk/management/commands/change_emails.py:32:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:35:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:44:14: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:48:13: E128 continuation line under-indented for visual indent
helpdesk/management/commands/change_emails.py:50:13: E128 continuation line under-indented for visual indent
```